contrib/index.rst

```diff
@@ -1,28 +1,116 @@
+to Python_ -- for both new and experienced contributors. It is :ref:`maintained
+<devguide>` by the same community that maintains Python.  We welcome your
+contributions to Python!
+
+We encourage everyone to contribute to Python. This guide should have
+everything you need to get started and be productive.  If you still have
+questions after reviewing the material in this guide, the `Core Python
+Mentorship`_ group is available to help you through the process.
+
+There are a number of ways to contribute including code, documentation, and
+triaging issues.  We've organized this guide to provide specifics based on the
+type of activity you'll be engaged in.
+
+
+Using this guide
+================
+
+We recommend reading this guide as needed. You can stop where you feel
+comfortable and begin contributing immediately without reading and
+understanding everything.  If you do choose to skip around this guide, be aware
+that it is written assuming preceding sections have been read so you may need
+to backtrack to fill in missing concepts and terminology.
 
-.. note::
-   This is a draft of a new organization for the devguide, turning it into a
-   Contributor's Guide.
+No matter what kind of contribution you'll be making, you should start with
+these common sections:
 
-[Much of the devguide home page will be here.  The Quick Start will be updated
-to distinguish code vs documentation contributions, and moved to those
-sections.  Other changes will be made to explain the different types of
-contribution and how to navigate the guide for your own type.]
+* :ref:`c_intro`
+* :ref:`c_project`
+* :ref:`c_gettingstarted`
 
+Then choose a path based on your type of activity:
+
+*[The original table on the devguide home had a fourth column for Core
+Developers. That made the table wider and more confusing. I don't think core
+team members need a quick intro path since they will have been through the
+devguide before.]*
+
+*[I haven't adjusted the links in the table yet other than to add a link to the
+major section at the top of each column.]*
+
+.. list-table::
+   :widths: 10 10 10
+   :header-rows: 1
+
+   * - Triaging
+     - Documentation
+     - Code
+   * -
+       :ref:`c_triage`
+
+       * :ref:`tracker`
+       * :ref:`triaging`
+       * :ref:`helptriage`
+       * :ref:`experts`
+       * :ref:`labels`
+       * :ref:`gh-faq`
+       * :ref:`triage-team`
+     -
+       :ref:`c_docs`
+
+       * :ref:`docquality`
+       * :ref:`documenting`
+       * :ref:`style-guide`
+       * :ref:`rst-primer`
+       * :ref:`translating`
+       * :ref:`devguide`
+     -
+       :ref:`c_code`
+
+       * :ref:`setup`
+       * :ref:`help`
+       * :ref:`pullrequest`
+       * :ref:`runtests`
+       * :ref:`fixingissues`
+       * :ref:`communication`
+       * :ref:`gitbootcamp`
+       * :ref:`devcycle`
+
+[The contents had been listed below here. It was very long, and I don't think
+it was helpful.  The left sidebar has the contents.]
```

Comment:
Hmm, I found it helpful: I often cmd+F on the homepage to find something, which won't work with a collapsed left sidebar.

It is a little unusual to have a large table-of-contents on a page, although it is out of the way at the bottom of the page, and not so big as to be a performance problem (like, we definitely wouldn't want the whole CPython TOC).

But it's not the end of the world to remove it if no one else wants it :)

Reply:
I think it is fine to leave at the very bottom.

Reply:
I can leave it.

Reply:
I put it back, but not as deep as before, so it's not so long.